### PR TITLE
Feature/435 upgrade node 20

### PR DESCRIPTION
**Pull request by RAnderson96**
update node to version 20 and npm audit fix
Issue: Scottish-Natural-Heritage/Deer-Online-Services#435

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "cy:open": "cypress open"
   },
   "engines": {
-    "node": "16.18.1"
+    "node": "20"
```
Comment (paulcoxsnh) [inline package.json:23]: `"node": ">=20.0.0"` - else will echo bad engine error all the time :)

**Review by paulcoxsnh — CHANGES_REQUESTED** [review]
Tiny change, see inline comment :)

File: package.json (hunk 1 above)
Comment (RAnderson96) [reply package.json:23]: Good catch!

Comment (paulcoxsnh) [reply package.json]: Resolved in https://github.com/Scottish-Natural-Heritage/cross-service-pages/pull/39/commits/d8e9f92b9786f531f2dfcc7768febad2b3bd6b7d.

**Review by paulcoxsnh — APPROVED** [review]
All good :)

**Pull request merged**